File: ios/ContentTransitionTextModule.swift
import ExpoModulesCore

public class ContentTransitionTextViewModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ContentTransitionTextView")

    View(ContentTransitionTextView.self) {
      Prop("value") { (view: ContentTransitionTextView, value: Double) in
      view.value = value
    }
  }
}
}

File: android/src/main/java/expo/modules/contenttransitiontext/ContentTransitionTextModule.kt
package expo.modules.contenttransitiontext

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ContentTransitionTextModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ContentTransitionTextView")

    View(ContentTransitionTextView::class) {
      Prop("value") { view: ContentTransitionTextView, value: Double ->
        view.setValue(value)
      }
    }
  }
}
